ci/install_cmake.sh

```diff
@@ -13,6 +13,6 @@ fi
 
 mkdir build
 cd build
-cmake ../.
+cmake -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ ../.
```

Comment:
huh, cmake doesn't like clang?

Reply:
I believe the issue is that Clang7 doesn't support string_view, which the newest CMake 3.15 requires. It does seem to work with GCC6 though, so I just opted to build CMake using GCC instead.